packages/preferences/src/common/json-pref-schema.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { JsonSchema, CombinedSchema } from "./json-schema";
+import { inject, named } from "inversify";
+import { ContributionProvider, ILogger } from '@theia/core/lib/common';
+import * as ajv from 'ajv';
+
+
+export const JsonSchemaContribution = Symbol("JsonSchemaContribution");
+export interface JsonSchemaContribution {
```

Comment:
It should be:
```ts
interface PreferenceContribution {
  readonly schema: {};
}
```

`JsonSchemaContribution` is over generic name does not relate to preferences at all